tests/association/test_positioning.py

```diff
@@ -21,6 +22,15 @@ def generate_mapped_pred(
         "page_num": page_num,
     }
 
+FILE_PATH = os.path.dirname(os.path.abspath(__file__))
+
+@pytest.fixture(scope="function")
+def bbox_token_page():
+    with open(os.path.join(FILE_PATH, "data/token_page/tokens.json"),
```

Comment:
it looks like the token page you use is massive -- generally for this kind of a test, you just need a dummy sample to test on, that could have a small number of samples that are very demonstrative. Looking at your test below there isn't a clear way for me to see it actually works

Reply:
Simplified the test token data by a lot to make it more obvious how tests work.